crates/ruma-state-res/src/lib.rs

```diff
@@ -167,24 +166,29 @@ fn separate<'a, Id>(
     state_sets_iter: impl Iterator<Item = &'a StateMap<Id>> + Clone,
```

Comment:
I I'm not mistaken the `Clone` bound is not needed anymore?

Reply:
Indeed. Fixed.